packages/apps/patterns/react-ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@dxos/react-uikit",
```

Comment:
```suggestion
  "name": "@dxos/react-ui",
```

Reply:
Rather, `@dxos/react-components`, I will make that change.